web_frontend/osmaxx/countries/models.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.gis.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .fields import InternalCountryFileField
+
+
+class Country(models.Model):
+    name = models.CharField(verbose_name=_('name'), max_length=100)
+    polyfile = InternalCountryFileField(verbose_name=_('polyfile'))
+    simplified_polygon = models.MultiPolygonField(verbose_name=_('simplified area'))
+
+    @property
+    def type_of_geometry(self):
+        """
+        Only needed for easier differentiation on the javascript side, has no real value otherwise.
```

Comment:
'discrimination' or more politically correct 'distinction' might be clearer terms than 'differentiation', which besides being a synonym for the former terms could also mean the computation of a mathematical function's derivative or becoming active in a wider field or more fields of business.